tests/tests/test_qtpy/test_collapsible_widget.py

```diff
@@ -8,6 +15,15 @@
 
 WIDGET_TITLE = "Title"
 
+## TODO Add tests for different layouts
+## TODO Add tests for removing widgets not in the container
```

Comment:
Do we need issues for these, so we keep track of TODO's in one place?

Reply:
I would say always have issues rather than TODOs